arelle/plugin/validate/EDINET/PluginValidationDataExtension.py

```diff
@@ -44,6 +44,20 @@
 
 _DEBIT_QNAME_PATTERN = regex.compile('.*(Liability|Liabilities|Equity)')
 
+STANDARD_TAXONOMY_URLS = frozenset((
```

Comment:
These aren't full URLs. `STANDARD_TAXONOMY_URL_PREFIXES` would be a better name.